### PR TITLE
2019 standards update

### PR DESCRIPTION
**Pull request by mayerm94**
Update models and fixtures to use QDM5.5, some cql logic used in fixtures cannot be calculated against QDM5.5 data and are skipped. Unskip these tests as part of https://jira.mitre.org/browse/BONNIE-2084

Pull requests into cqm-execution require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-2083
- [x] Internal ticket links to this PR https://jira.mitre.org/browse/BONNIE-2083
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases NA
- [x] Tests have been run locally and pass NA

**Reviewer 1:**

Name: @losborne 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases (skipped tests with future task drops coverage but is expected)
- [x] You have tried to break the code

**Reviewer 2:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: spec/helpers/results_helpers_spec.js [hunk 1]
```diff
@@ -51,7 +51,8 @@ describe('ResultsHelpers', () => {
     });
 
     describe('pretty statement results when requested', () => {
-      it('for CMS107v6 correctly', () => {
+      xit('for CMS107v6 correctly', () => {
+        // TODO: Find another measure to use. PrincipalDiagnosis is no longer a QDM attribute and is used in this measure logic
```
Comment (losborne) [inline spec/helpers/results_helpers_spec.js]: I don't think this test necessarily has to test the pretty result for principal diagnosis--could we just swap out the clause with another one on the same measure?

Comment (losborne) [reply spec/helpers/results_helpers_spec.js]: On the other hand, since the tests below do need to be updated anyway, I could see doing them all in one batch.  Is there a future task to unskip these tests? Does that need to be done before any other significant work is done on cqm-execution? @jbradl11

**Comment by jbradl11** [comment]
Closing because 2019-standards-update is becoming a feature branch.
